apps/web/src/hooks/useSharingActions.ts

```
@@ -13,7 +13,13 @@ import { sharingStore, type VerifiedContact } from '../stores/sharing.store';
 import { useCommandRunner } from './useCommandRunner';
 
 /** Which call is in flight, or `null` when the sharing surface is idle. */
-export type SharingCommand = 'read' | 'importContact' | 'grant' | 'revoke' | 'downgrade';
+export type SharingCommand =
+  | 'read'
+  | 'importContact'
+  | 'grant'
+  | 'revoke'
+  | 'downgrade'
+  | 'createInviteLink';
```

ESLint

[error] Replace `|·'read'⏎··|·'importContact'⏎··|·'grant'⏎··|·'revoke'⏎··|·'downgrade'⏎·` with `'read'·|·'importContact'·|·'grant'·|·'revoke'·|·'downgrade'`

(prettier/prettier)